upf/upf_api.c

```diff
@@ -977,6 +977,73 @@ vl_api_upf_get_node_id_t_handler (vl_api_upf_get_node_id_t * mp)
   vl_api_send_msg (reg, (u8 *) rmp);
 }
 
+/* API message handler */
+static void
+vl_api_upf_tdf_ul_table_t_handler (vl_api_upf_tdf_ul_table_t * mp)
+{
+  int rv = 0;
+  vl_api_registration_t *reg;
+  reg = vl_api_client_index_to_registration (mp->client_index);
+  if (!reg)
+    return;
+
+  vl_api_upf_tdf_ul_table_reply_t *rmp = NULL;
+  upf_main_t *sm = &upf_main;
+  fib_protocol_t fproto = mp->is_ipv6 ? FIB_PROTOCOL_IP6 : FIB_PROTOCOL_IP4;
+
+  u8 len = 0;
+  u32 ii;
```

Comment:
In order to be consistent with the rest of VPP codebase, we declare all of the vars on top of the `{ ... }` block